brjs-core-tests/src/test/java/org/bladerunnerjs/spec/aliasing/AliasModelTest.java

```diff
@@ -1,332 +0,0 @@
-package org.bladerunnerjs.spec.aliasing;
-
-import java.io.IOException;
-
-import org.bladerunnerjs.aliasing.aliases.AliasesFile;
-import org.bladerunnerjs.api.App;
-import org.bladerunnerjs.api.Aspect;
-import org.bladerunnerjs.api.Blade;
-import org.bladerunnerjs.api.Bladeset;
-import org.bladerunnerjs.api.aliasing.AmbiguousAliasException;
-import org.bladerunnerjs.api.aliasing.IncompleteAliasException;
-import org.bladerunnerjs.api.aliasing.NamespaceException;
-import org.bladerunnerjs.api.aliasing.UnresolvableAliasException;
-import org.bladerunnerjs.api.aliasing.aliasdefinitions.AliasDefinitionsFile;
-import org.bladerunnerjs.api.memoization.MemoizedFile;
-import org.bladerunnerjs.api.model.exception.request.ContentFileProcessingException;
-import org.bladerunnerjs.api.spec.engine.SpecTest;
-import org.bladerunnerjs.model.AssetContainer;
-import org.bladerunnerjs.utility.FileUtils;
-import org.junit.Before;
-import org.junit.Test;
-
-import com.ctc.wstx.exc.WstxValidationException;
-
-public class AliasModelTest extends SpecTest {
```

Comment:
No, but you're right they should at least be converted to test the `AliasingUtility` class or similar. Will look at this today.